modules/ob-release-notes-1-5-1.adoc

```diff
@@ -0,0 +1,24 @@
+// This module is included in the following assemblies:
```

Comment:
Can we just mention that it is included in the following assembly, instead of assemblies?